app/assets/javascripts/admin/addon/components/admin-config-areas/components.gjs

```diff
@@ -45,19 +93,444 @@ export default class AdminConfigAreasComponents extends Component {
-            @component={{true}}
-            @openModal={{this.installModal}}
-          />
-        </:specialCard>
-      </ThemesGrid>
+    <DPageSubheader
+      @titleLabel={{i18n
+        "admin.config_areas.themes_and_components.components.title"
+      }}
+      @descriptionLabel={{i18n
+        "admin.config_areas.themes_and_components.components.description"
+      }}
+    >
+      <:actions as |actions|>
+        <actions.Primary
+          disabled={{this.loading}}
+          @label="admin.config_areas.themes_and_components.components.install"
+          @action={{this.installModal}}
+        />
+      </:actions>
+    </DPageSubheader>
+    <div class="container">
+      {{#if this.hasComponents}}
+        <div class="d-admin-filter">
+          <div
+            class="admin-filter__input-container admin-config-components__name-filter"
+          >
+            <FilterInput
+              placeholder={{i18n
+                "admin.config_areas.themes_and_components.components.search_components"
+              }}
+              @filterAction={{this.onNameFilterChange}}
+              class="admin-filter__input"
+            />
+          </div>
+
+          <label class="admin-config-components__status-filter">
+            {{i18n
+              "admin.config_areas.themes_and_components.components.filter_by"
+            }}
+            <DSelect
+              @value="all"
+              @includeNone={{false}}
+              @onChange={{this.onStatusFilterChange}}
+              as |select|
+            >
+              {{#each STATUS_FILTER_OPTIONS as |option|}}
+                <select.Option @value={{option.value}}>
+                  {{i18n option.label}}
+                </select.Option>
+              {{/each}}
+            </DSelect>
+          </label>
+        </div>
+      {{/if}}
+      <ConditionalLoadingSpinner @condition={{this.loading}}>
+        {{#if this.components.length}}
+          <table class="d-admin-table component-list">
+            <thead>
+              <th>{{i18n
+                  "admin.config_areas.themes_and_components.components.name"
+                }}</th>
+              <th>{{i18n
+                  "admin.config_areas.themes_and_components.components.used_on"
+                }}</th>
+              <th>{{i18n
+                  "admin.config_areas.themes_and_components.components.enabled"
+                }}</th>
+              <th></th>
+            </thead>
+            <tbody>
+              {{#each this.components as |comp|}}
+                <ComponentRow @component={{comp}} @refresh={{this.load}} />
+              {{/each}}
+            </tbody>
+          </table>
+        {{else}}
+          {{#if this.hasComponents}}
+            {{i18n
+              "admin.config_areas.themes_and_components.components.no_components_found"
+            }}
+          {{else}}
+            {{i18n
+              "admin.config_areas.themes_and_components.components.no_components"
```

Comment:
Do you think we can use our            <AdminConfigAreaEmptyList /> component? Not sure if it will fit the design.

This is how it looks on permalinks

<img width="1378" alt="Screenshot 2025-04-08 at 11 59 41 am" src="https://github.com/user-attachments/assets/2c51cefb-bd1e-4892-8c05-2c9d24c3b8ae" />